functions/validator/requirements.txt

```diff
@@ -14,3 +14,5 @@ packaging==16.8
 setuptools==35.0.2
 appdirs==1.4.3
 pyparsing==2.2.0
+datetime
+git+git://github.com/mozilla-iam/cis.git
```

Comment:
yeah it's best just to leave anything that's natively part of the runtime out to avoid conflicts.